src/community/advanced-data-card.tsx

```diff
@@ -201,7 +208,7 @@ type AllowedExtra =
 type TextAs = 'h1' | 'h2' | 'h3' | 'h4';
 
 interface AdvancedDataCardProps {
-    onPress?: () => void;
+    href?: string | undefined;
```

Comment:
The change from onPress to href is so that on the screen reader it can be read as a link instead of a button.

Reply:
Hi @atabel,
Is something pending on our side?

Reply:
We can't make a major version by this breaking change, we need to support `onPress` + `href`, and we also can include `to` to have parity with the rest of the cards. But we need to keep onPress because in master is already existing and there are teams using this prop.

@EduardoLafuente @BrunoGuimaraes103